ci_scripts/generate_docs.py

```diff
@@ -0,0 +1,27 @@
+"""Script to generate documentation using pdoc"""
+from subprocess import check_call
+
+from utils.definitions import MODULES_TO_DOCUMENT, DOCUMENTATION_DIR_PATH
+
+
+def main() -> None:
+    """Triggers building the documentation.
+
+    Modules the the documentation are built for and the documentation
+    destination path can be set in the utils.definitions config file.
+    """
+    command_list = [
+        "pdoc", "--output-dir", f'{DOCUMENTATION_DIR_PATH}',
+        "--force", "--config", "show_type_annotations=True"
+    ]
+
+    for module_name in MODULES_TO_DOCUMENT:
+        command_list.append("--html")
+        command_list.append(f'{module_name}')
+
+    print('Creating pdoc documentation.')
```

Comment:
prefer `logging` package methods such as `info, warn, error`   for logging information as it gives more flexibility around where and what messages are shown to user

Reply:
Sure 👍